pkg/roachprod/roachprod.go

```diff
@@ -1329,9 +1384,20 @@ func destroyCluster(cld *cloud.Cloud, l *logger.Logger, clusterName string) erro
 		l.Printf("Destroying cluster %s with %d nodes", clusterName, len(c.VMs))
 	}
 
+	deleteClusterConfig(clusterName, l)
+
 	return cloud.DestroyCluster(l, c)
 }
 
+// deleteClusterConfig deletes the prometheus instance cluster config. Any error is logged and ignored.
+func deleteClusterConfig(clusterName string, l *logger.Logger) {
+	if err := promhelperclient.NewPromClient().DeleteClusterConfig(context.Background(),
+		envutil.EnvOrDefaultString(prometheusHostUrlEnv, defaultPrometheusHostUrl),
+		clusterName, false, l); err != nil {
+		l.Errorf("deleting cluster config failed: %v", err)
```

Comment:
These will be hard to observe. We do plan to also extend `roachprod gc` to delete the dangling resource, right?

Reply:
makes sense. should I rather return the error from here for now?

Reply:
I don't think we want to fail `Delete` (and transitively the test) if the above cleanup step fails. Let's leave as is and make sure `roachprod gc` is extended with this logic.